hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -272,10 +268,9 @@
   private final Cache<Long, Long> executedRegionProcedures =
       CacheBuilder.newBuilder().expireAfterAccess(600, TimeUnit.SECONDS).build();
 
-  // Cache flushing
-  protected MemStoreFlusher cacheFlusher;
+  private MemStoreFlusher cacheFlusher;
 
-  protected HeapMemoryManager hMemManager;
+  private HeapMemoryManager hMemManager;
```

Comment:
What version we targetting this patch against? HRS is LimitedPrivate for Tools. Wonder if any tools using these data members. For 3.0.0 or 2.3.0?

Reply:
I'm thinking this is for master and branch-2. Any further and these kind of refactor/cleanup patches end up not being "back ports" but pre-branch rewrites. I could be persuaded, but you'll have to be pretty decorated in Debate Club.

Reply:
I suppose I should go read the fine-print re: `IA.LimitedPrivate(Tools)`...

Reply:
What did you figure reading fine-print?

Reply:
I see you added it on the end.